applications/blockchainia.md

```diff
@@ -0,0 +1,139 @@
+
+This project will involve creating an api for interactions between our off-chain game server and parachain. Custom pallets will relay real-time in-game events between the blockchain and off-chain worker. Off-chain servers must be authorized through a DAO that runs our parachain, which will ensure the integrity of those who host online competative gameplay. Community members who run servers will earn a portion of each players pay-to-play fee, a portion of which will pay the parachain overhead incurred by the server. 
+
+This project will allow us to test our parachain and online multiplayer infrastructure to prepare for market research.
+
+With a successful initial MVP, we can perform market research in the nearby Business Research Center at the University of Pittsburgh. This research will allow us to create pallettes that encourage consumer engagement and interaction. With the data output from these studies and focus groups we can publish estimated potential earnings to entice other developers to use our infrastructure and distribution model. 
+
+While there are other gaming projects in the ecosystem, ours differs in various ways. Our pay-to-play gaming infrastructure will funnel our currency from end-users, through server owners which pay a fee to our parachain to connect to the network and access the ledger. Events unique to our architecture include spawning AI enemies from ink! smart contract created NFTs. These enemies will earn in-game currency for its owner by completing in-game tasks, like eliminating players. This encourages users not only to purchase an NFT to play the game, but also to earn back the tokens they spend to play the game. Each AI environment enemy in our game will be owned as an NFT on our blockchain. This will drive our economy and community. 
+
+We will create free-to-use Unity assets that compliment our tech stack to allow independent developers to duplicate our model and optionally distribute their games via our distribution architecture or use our reputation based, community moderated online-multiplayer infrastructure.
+
+## Project Details
+
+### Architecture Diagram
+![Blockchainia Architecture](https://github.com/eca20/Blockchainia-Pitch/blob/main/Blockchainia.drawio.png?raw=true "Blockchainia Architecture")
```

Comment:
I struggle to understand the diagram, could you add some explanations along with it?
For example:

- I'm not sure if the arrows represent _dependencies_ or _data flow_
- I'm unclear on the differences between _DAO owned server_ and _Game Distribution Node_
- I'm confused whether the green, little _DAO owned server_s within the _Parachain_ represent the same servers as the _DAO owned server_s that are more detailed outside of the _Parachain_

Reply:
The arrows represent dataflow. One arrow represents an interaction between the two entities in both directions. We will use a server-authoritative model to reconcile inputs and the timing of events from multiple game clients. Custom server configurations will draw specific player archetypes. These "regulars" make up the servers community, which will lead to unique "regions", rules, and play styles among the various cliques that emerge in the community. Each region will be able request unique features to further enhance their gameplay and enable further configurations to customize gameplay. These requested features will be added to the backlog of our development team. Clients will login with their wallet and connect to a game server. The game server will relay in-game event information to the blockchain. A game client labeled as an Admin has special status in the DAO, given be the server owner to allow in-game privileges.  Admins can access commands that allow them to penalize or otherwise moderate player behavior, as well as combat cheating/hacking. In the future, we hope to host a library of games that interact with our blockchain through specialized nodes that host the library of games that interact with our infrastructure for download. The game distribution node is not something we are not currently working on, but would be the final step in decentralizing video game distribution, replacing retailers and online stores such as Steam, the App Store, and Xbox marketplace, as small teams of developers can release straight to consumer, and earn from selling the NFT collections that enable users to play their games.